### ISSUE TITLE
can we get screenshots?

### ISSUE THREAD

**Issue by maitra** [hydrargyrum/lierre#1]

**Comment by hydrargyrum** [comment]
Good idea. By any chance, do you know where I could find dummy emails to use for demoing? Else I will roll some random email generator.

**Comment by maitra** [comment]
Sorry, I do not know. Random email generator would be good. Btw, any chance that you will have local folder support (basically a sidepaneor so) such as in neomutt?

**Comment by hydrargyrum** [comment]
@maitra sorry for taking some time, I added one screenshot in the readme, I will add more later.

> Btw, any chance that you will have local folder support (basically a sidepaneor so) such as in neomutt?

What is it exactly?

Lierre's main view shows a search bar, the tags list in a sidebar and the threads list matching the search. IMAP folders are mapped to local folders using `mbsync` for example. [`notmuch search`](https://notmuchmail.org/manpages/notmuch-search-terms-7/) (and by extension, Lierre's search bar too) allows to search by folder (for example: `folder:.foo`). However, there is no way to view the folder tree within Lierre (there's a widget for debugging only though). Is it what you would want?

**Comment by maitra** [comment]
> @maitra sorry for taking some time, I added one screenshot in the readme, I will add more later.
> 
> > Btw, any chance that you will have local folder support (basically a sidepaneor so) such as in neomutt?
> 
> What is it exactly?
> 
> Lierre's main view shows a search bar, the tags list in a sidebar and the threads list matching the search. IMAP folders are mapped to local folders using `mbsync` for example. [`notmuch search`](https://notmuchmail.org/manpages/notmuch-search-terms-7/) (and by extension, Lierre's search bar too) allows to search by folder (for example: `folder:.foo`). However, there is no way to view the folder tree within Lierre (there's a widget for debugging only though). Is it what you would want?

Thanks for all your hard work! The screenshot looks really nice. It would be nice to have a sidepane or such list of folders with number of unread/total e-mails, as in neomutt. I am thinking of something like:

Inbox (2/465)
drafts (2/4)
personal (0/34)
work (0/56)

Sort of like is available on neomutt or sylpheed. Does this help? Thanks again for the screenshot, it looks really nice!!

**Comment by hydrargyrum** [comment]
> It would be nice to have a sidepane or such list of folders with number of unread/total e-mails, as in neomutt. I am thinking of something like:
>
> Inbox (2/465)
drafts (2/4)
personal (0/34)
work (0/56)

I added a screenshot of the threads list and the tags list:

https://raw.githubusercontent.com/hydrargyrum/lierre/f4b5e83f879f0b47e778bf1c47db2c66dd532d2e/2019-12-22-163338_1300x897_scrot.png

However it only shows the number of unread mails, not total mails, it could be an important information to show indeed, good idea.

Lierre, like notmuch, prefers "tags" over "folders".
Keep in mind Lierre is still in alpha stage. It will not cause mail destruction but has some performance issues, and many features are still left [to be implemented](https://github.com/hydrargyrum/lierre/blob/master/TODO.md), possibly with the existing plugin system.

**Comment by maitra** [comment]
Wonderful. Thank you very much! I am looknig forward to using it and perhaps packaging it for Fedora (which is the software that I use) so I hope that you will be able to implement some of the features and move it from alpha (I doubt that they will allow for an alpha software). However, I don't think that the performance issues are that severe or I have not found it to be so, so thanks again for the software.

Btw, can the tags list also have subtags? What I am thinking is that under the tag "work", I could have two subcategories? And that could be listed also perhaps? Perhaps it is already there, perhaps not, and it is not worth it, but I thought that i would ask. Thanks again!

**Comment by hydrargyrum** [comment]
> Btw, can the tags list also have subtags? What I am thinking is that under the tag "work", I could have two subcategories? And that could be listed also perhaps? Perhaps it is already there, perhaps not, and it is not worth it, but I thought that i would ask. Thanks again!

Per se, notmuch is quite minimalistic and doesn't have a subtag notion. But nothing prevents you from defining, let's say, ":" as a subtag separator and then searching [`tag:/^mytag:.*/`](https://notmuchmail.org/manpages/notmuch-search-terms-7/) to search all subtags of `mytag`. This should be included builtin in Lierre interface to make it more user-friendly. I'll think about it.

**Comment by maitra** [comment]
Thanks very much! Yes, that would be nice. 

Looking forward to further updates on lierre. Btw, you probbaly need an icon for lierre:-)

**Comment by maitra** [comment]
One more question: how does lierre display and handle attachments like pdf, word, etc? Also sending attachments using MIME?
